libs/ledgerjs/packages/types-live/src/derivation.ts

```diff
@@ -32,4 +32,5 @@ export type DerivationMode =
   | "stacks_wallet"
   | "icon"
   | "ton"
-  | "aptos";
+  | "aptos"
+  | "minabip44h";
```

Comment:
bip44 - h ?

What is h by curiosity ?

Reply:
Looks like a mistake tbh, no reason I can think of. bip44 hardened doesn't apply here anyway. I have renamed the derivationMode to `bip44`